src/Vts.Test/IO/CustomBinaryStreamWriterOfTTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework;
+using Vts.IO;
+
+namespace Vts.Test.IO
+{
+    /// <summary>
+    /// Stream finder tests
+    /// </summary>
+    [TestFixture]
+    public class CustomBinaryStreamWriterOfTTests
+    {
+        /// <summary>
+        /// list of temporary files created by these unit tests
+        /// </summary>
+        readonly List<string> listOfTestGeneratedFolders = new List<string>()
+        {
+            "results_CustomBinaryStreamWriterOfTTests",
+        };
+
+        /// <summary>
+        /// clear all generated folders and files
+        /// </summary>
+        [OneTimeSetUp]
+        [OneTimeTearDown]
+        public void clear_folders_and_files()
+        {
+            foreach (var folder in listOfTestGeneratedFolders)
+            {
+                Directory.Delete(folder, recursive: true);
+            }
+        }
+
+        /// <summary>
+        /// Validate writing text to a stream
+        /// </summary>
+        [Test]
+        public void validate_does_not_throw_when_no_folder_exists()
```

Comment:
Method name starts with uppercase: Validate_does_not_throw_when_no_folder_exists()

Reply:
Does it make sense to have a different standard for tests? Not sure there's a benefit of a test being capitalized for just the first word. Most of our tests in this project start with lowercase, at the moment.

Reply:
We have been updating them as we go along, we're just trying to follow a standard with all of the code going forward.